src/test/java/pages/LoginPage.java

```diff
@@ -0,0 +1,43 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+public class LoginPage extends BasePage {
+    BasePage basePage = new BasePage();
+    public String myEmail = "victor.colodzei@testpro.io";
+    public String myPassword = "hyqsis-1viQni-nogjyx";
+    public String demoEmail = "demo@class.com";
+    public String demoPassword = "te$t$tudent";
+    public String wrongPassword = "te$t$tuden";
+    public String emptyPassword = "";
+
+    // locators
+    By passwordField = By.cssSelector("[type='password']");
+    By emailField = By.xpath("//input[@type='email']");
+    By submitBtn = By.cssSelector("button[type='submit']");
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```